util/libcrypto.num

```diff
@@ -4342,6 +4342,21 @@ OSSL_STORE_INFO_set0_NAME_description   4284	1_1_1	EXIST::FUNCTION:
 OSSL_STORE_INFO_get1_NAME_description   4285	1_1_1	EXIST::FUNCTION:
 OSSL_STORE_do_all_loaders               4286	1_1_1	EXIST::FUNCTION:
 OSSL_STORE_LOADER_get0_engine           4287	1_1_1	EXIST::FUNCTION:
-OPENSSL_fork_prepare                    4288	1_1_1	EXIST:UNIX:FUNCTION:
-OPENSSL_fork_parent                     4289	1_1_1	EXIST:UNIX:FUNCTION:
-OPENSSL_fork_child                      4290	1_1_1	EXIST:UNIX:FUNCTION:
```

Comment:
Not sure why you're moving those...  no biggie, since these symbols are not part of any release yet, but essentially bad form, gives me hickups...